### PR TITLE
Video-intelligence: Regeneration

### PR DESCRIPTION
**Pull request by landrito**

### CODE REVIEW

File: packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js [hunk 1]
```diff
@@ -288,4 +288,4 @@ function VideoIntelligenceServiceClientBuilder(gaxGrpc) {
 }
 module.exports = VideoIntelligenceServiceClientBuilder;
 module.exports.SERVICE_ADDRESS = SERVICE_ADDRESS;
-module.exports.ALL_SCOPES = ALL_SCOPES;
\ No newline at end of file
+module.exports.ALL_SCOPES = ALL_SCOPES;
```
Comment (swcloud) [inline packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js]: This needs to be fixed in toolkit.

File: packages/video-intelligence/src/v1beta1/index.js [hunk 2]
```diff
@@ -31,4 +31,4 @@ v1beta1.GAPIC_VERSION = '0.1.0';
 v1beta1.SERVICE_ADDRESS = videoIntelligenceServiceClient.SERVICE_ADDRESS;
 v1beta1.ALL_SCOPES = videoIntelligenceServiceClient.ALL_SCOPES;
 
-module.exports = v1beta1;
\ No newline at end of file
+module.exports = v1beta1;
```
Comment (swcloud) [inline packages/video-intelligence/src/v1beta1/index.js]: This needs to be fixed in toolkit.

File: packages/video-intelligence/src/index.js [hunk 3]
```diff
@@ -57,4 +57,4 @@ extend(v1beta1Protos, gaxGrpc.load([{
 module.exports = videoIntelligenceV1beta1;
 module.exports.types = v1beta1Protos;
 module.exports.v1beta1 = videoIntelligenceV1beta1;
-module.exports.v1beta1.types = v1beta1Protos;
\ No newline at end of file
+module.exports.v1beta1.types = v1beta1Protos;
```
Comment (swcloud) [inline packages/video-intelligence/src/index.js]: This needs to be fixed in toolkit.

File: packages/video-intelligence/package.json [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
-  "name": "@google-cloud/videointelligence",
+  "name": "@google-cloud/video-intelligence",
   "version": "0.1.0",
   "author": "Google Inc",
   "description": "Google Cloud Video Intelligence API client for Node.js",
   "main": "src/index.js",
   "files": [
     "src",
     "AUTHORS",
+    "CONTRIBUTORS",
```
Comment (swcloud) [inline packages/video-intelligence/package.json]: Don't need this hand edit.

File: packages/video-intelligence/package.json [hunk 5]
```diff
@@ -19,22 +20,24 @@
...
+    "google video intelligence",
+    "video intelligence",
+    "video",
     "google video-intelligence",
     "video-intelligence",
     "Google Cloud Video Intelligence API"
   ],
   "dependencies": {
-    "google-proto-files": "^0.12.0",
+    "extend": "^3.0.0",
```
Comment (swcloud) [inline packages/video-intelligence/package.json]: Don't need hand edits in this section.

File: packages/video-intelligence/package.json [hunk 6]
```diff
@@ -19,22 +20,24 @@
     "google cloud platform",
     "google cloud",
     "cloud",
+    "google video intelligence",
```
Comment (swcloud) [inline packages/video-intelligence/package.json]: This may be added to toolkit

File: packages/video-intelligence/src/v1beta1/index.js [hunk 7]
```diff
@@ -27,7 +27,7 @@ function v1beta1(options) {
   return videoIntelligenceServiceClient(gaxGrpc);
 }
 
-v1beta1.GAPIC_VERSION = '0.7.1';
+v1beta1.GAPIC_VERSION = '0.1.0';
```
Comment (swcloud) [inline packages/video-intelligence/src/v1beta1/index.js]: This needs to be '0.7.1'

File: packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js [hunk 8]
```diff
@@ -35,7 +35,7 @@ var SERVICE_ADDRESS = 'videointelligence.googleapis.com';
 
 var DEFAULT_SERVICE_PORT = 443;
 
-var CODE_GEN_NAME_VERSION = 'gapic/0.7.1';
+var CODE_GEN_NAME_VERSION = 'gapic/0.1.0';
```
Comment (swcloud) [inline packages/video-intelligence/src/v1beta1/video_intelligence_service_client.js]: This needs to be '0.7.1'

**Comment by landrito** [comment]
@swcloud Made this simply the generated packages including the docs. I'll address the hand-edits in toolkit.

One hand-edit i needed to do was changing the path and packagename to video-intelligence.

**Comment by swcloud** [comment]
@landrito LGTM except the GAPIC version number. @stephenplusplus may also need to approve it before he merges to the master.

**Pull request closed without merging**